pages/data-design/groups.mdx

```diff
@@ -0,0 +1,132 @@
+---
+layout: 'guide'
+title: 'Intro to Group analytics'
+---
+
+import Link from '../../components/Link';
+
+When doing product analytics you may find yourself in a position when you need to analyze behavior of a group of users, or you need to group
+different actions together into a group.
+
+There are two main ways of group analytics that we are going to look at in this article:
+
+- Grouping users
+- Grouping actions
+
+#### Groups concepts
```

Comment:
I think this section is unnecessarily early in this doc. As reader learning about why I'd use Group Analytics, the sections about Grouping Users and Grouping Events are more valuable to learn the high level idea. 

I'd suggest:
1. moving this to the bottom, 
2. having it higher level in the page tree (## instead of ####)
3. renaming to "Group terminology across platforms"

Reply:
```suggestion
## Groups terminology across platforms
```